web/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "serve": "npm run start",
-    "start": "docusaurus start --port 3005",
+    "start": "docusaurus start --port 3006",
```

Comment:
just made a small change to the port docusaurus runs on so it doesn't conflict with a docker daemon service running locally